### ISSUE TITLE
Nullable columns issues

### ISSUE THREAD

**Issue by aadim** [ClickHouse/clickhouse-odbc#43]
While querying a table with nullable columns from Tableau using 11-07 build I have Syntax error. The query is 

`select platforms from gamora.params where platforms is null limit 1`

Tableau wraps it and generates the following query:

```
SELECT `Custom_SQL_Query`.`platforms` AS `platforms`
FROM (
  select platforms from gamora.params where platforms is null limit 1
) `Custom_SQL_Query`
GROUP BY `platforms`
```

that leads to the error. The query is OK and according to Wireshark dump CH replues HTTP OK with data.

` :) SELECT `Custom_SQL_Query`.`platforms` AS `platforms`
    FROM (
      select platforms from gamora.params where platforms is null limit 1
    ) `Custom_SQL_Query`
    GROUP BY `platforms`

SELECT `Custom_SQL_Query`.`platforms` AS `platforms`
  FROM (
        select platforms
          from gamora.params
         where platforms is null
         limit 1
       ) `Custom_SQL_Query`
 GROUP BY `platforms`

┌─platforms─┐
│        \N │
└───────────┘

Ok. 1 row in set. Elapsed: 0.017 sec. Processed: 0.0  rows, 0.0B (0.0  rows/s, 0.0B/s)
`

`show create table gamora.params

CREATE TABLE gamora.params ( user_id Int64,  date Date,  dt DateTime,  p1 Nullable(Int32),  platforms Nullable(Int32),  max_position Nullable(Int32),  vv Nullable(Int32),  city Nullable(String),  third_party Nullable(Int8),  mobile_tablet Nullable(Int8),  mobile_phone Nullable(Int8),  desktop Nullable(Int8),  web_mobile Nullable(Int8),  tv_attach Nullable(Int8),  smart_tv Nullable(Int8),  subsite_id Nullable(Int32),  view_in_second Nullable(Int32),  view_in_second_presto Nullable(Int32)) ENGINE = MergeTree(date, user_id, 8192) 
`

**Comment by MaximKhorev** [comment]
+1

**Comment by proller** [comment]
Problem still actual? (on latest version of clickhouse and odbc driver)

**Comment by alexey-milovidov** [comment]
It is still relevant.

**Comment by traceon** [comment]
Tried the following with the latest release of the ODBC driver (`v1.1.1.20191108`):

```
> clickhouse-client --query "CREATE TABLE nullable_col_issue_tbl ( nullable_col Nullable(Int32) ) ENGINE = Memory"
> clickhouse-client --query "INSERT INTO nullable_col_issue_tbl VALUES ( NULL )"
> clickhouse-client --query "SELECT nullable_col from nullable_col_issue_tbl WHERE nullable_col IS NULL LIMIT 1"
\N
```

then executed that same SELECT query in Tableau in Windows, and got this:
<img width="1680" alt="Screenshot 2019-11-12 at 22 42 41" src="https://user-images.githubusercontent.com/10261534/68700682-ba7dce00-059e-11ea-9604-420344f7c9c4.png">

Please close this issue, if the above can be considered an indication that this issue is fixed, at least starting `v1.1.1.20191108`.
